source/collections/Vector.ooc

```diff
@@ -98,6 +99,14 @@ HeapVector: class <T> extends Vector<T> {
 	__destroy__: func {
 		gc_free(this _backend)
 	}
+
+		operator [] (index: Int) -> T {
+			this _backend[index]
+		}
+
+		operator []= (index: Int, item: T) {
```

Comment:
One last thing: Before `this _backend[index] = item`, do this:
```ooc
if (T inheritsFrom?(Object)) {
	old := this[index] as Object
	old free()
}
```
or else whatever is already in `_backend[index]` will leak.